### PR TITLE
AST-1201 - test case for button border color, border radius, border width

### PR DESCRIPTION
**Pull request by bsfsnehal**
### Description

Added E2E test for button border color, border radius, border width
### Screenshots

https://share.getcloudapp.com/GGu4AvLN
### How has this been tested?

Execution- npm run test:e2e:interactive — tests/e2e/specs/customizer/global/button/button-border.test.js
### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/global/button/button-border.test.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+				left: 5,
+			},
+		};
+		await setCustomize( borderWidth );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( 'form[CLASS*="wp-block-search__"].wp-block-search .wp-block-search__inside-wrapper .wp-block-search__button' );
+		await expect( {
+			selector: 'form[CLASS*="wp-block-search__"].wp-block-search .wp-block-search__inside-wrapper .wp-block-search__button',
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/global/button/button-border.test.js:19]: Could you please add all selectors and verify to all:- https://share.bsf.io/rRu1QNO1

Comment (bsfsnehal) [reply tests/e2e/specs/customizer/global/button/button-border.test.js:19]: okay

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/global/button/button-border.test.js:19]: https://share.getcloudapp.com/rRu1RkkP - Also add test case for these option

Comment (bsfsnehal) [reply tests/e2e/specs/customizer/global/button/button-border.test.js]: Added multiple selectors.

File: tests/e2e/specs/customizer/global/button/button-border.test.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+				title: 'buttonBorder',
+			} );
+			await insertBlock( 'Buttons' );
+			await page.keyboard.type( 'Login' );
+			ppStatus = await publishPost();
+		}
+		await page.goto( createURL( 'buttonBorder' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '#block-2 .wp-block-search__button, .wp-block-button .wp-block-button__link' );
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/global/button/button-border.test.js:28]: **#block-2 .wp-block-search__button** this selector not used for search please verify,

**Review by SimranChougule1 — APPROVED** [review]

**Review by SimranChougule1 — APPROVED** [review]

**Review by GauravKhupse — APPROVED** [review]

**Pull request merged**